src/app/navbar/navbar.component.ts

```diff
@@ -67,5 +68,16 @@ export class NavbarComponent implements OnInit {
     toggleInfokit() {
         this.infoPop.emit();
     }
+
+    editUser(): void {
+      const dialogConfig = new MatDialogConfig();
+      dialogConfig.data = {
+        profilePicture: this.proPic,
+        username: this.username
+      };
+      dialogConfig.width = '600px';
```

Comment:
Is there no other way to mention this.. you should be using ts files for css properties as last resort..

Reply:
@shreyans29 last resort would be to make a config file. But with having only this entry in it.